cardigan_new/stories/global/spacing/spacing.mdx

```diff
@@ -0,0 +1,40 @@
+import { Meta } from '@storybook/addon-docs';
+import { Spacing } from './spacing.stories.tsx';
+
+<Meta title="Global/Spacing" />
+
+# Spacing
+
+Having a consistent set of spacings between and within components lends a consistent look and feel to the site.
+
+It also enables us to make systematic decisions without being reliant on visual designs any time a new component is created and needs to be placed on a page.
+
+## Section spacing
+
+We define 'Sections' as the top-level elements of the `ContentPage`
+
+- `PageHeader`
+- `Body`
+- `children`
+- `Contributors`
+- `RelatedContent`
+
+Each of these is wrapped in a `<SpacingSection />`, which adds padding at the bottom in order to space them apart.
+
+## Component spacing
+
+Components _within_ these sections are wrapped in a `<SpacingComponent />`. This adds margin to the top to space the components out.
+
+We don't apply this top margin to the first `<SpacingComponent />`, since we don't want to add it to the bottom-margin added by a preceding `<SpacingSection />`.
+
+We handle spacing in body copy and `<SpacingComponent />`s in a similar way[^1]. The main difference being that in body copy we use `em` units.
+
+These are units that are relative to the size of the type, so this enables us to keep the spacing hierarchy consistent when we change the font size (e.g. at different breakpoints) without having to change an absolute pixel value for spacing at the same time.
+
+## Demo
+
+<Spacing />
+
+[^1]: [Axiomatic CSS and Lobotomized Owls](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls)
+
+[Edit this Readme on GitHub](https://github.com/wellcometrust/wellcomecollection.org/blob/main/cardigan/stories/global/spacing/spacing.stories.mdx)
```

Comment:
Should this be cardigan_new?

Reply:
I think I reasoned that at the point that this will be live it will live at `/cardigan/` rather than `/cardigan_new/` and (since nobody ever really updates them in GitHub) it seemed unlikely they would be updated via GitHub before that point.

That said, I guess it would be a simple find/replace along with the other instances of 'cardigan_new', so probably wouldn't hurt to do what you're suggesting. I'll update it.

Reply:
Agree with you too, so whichever is good. It'd be great if there was a way to "link to self" but I think I'm asking to much of markdown files.